README.md

```diff
@@ -1,33 +1,96 @@
-filenaming by exposing the `filesystemName` function in the template view class, which Nota-server will
-poll to see if it yields a usable filename.
+````
+node dist/nota.js --template=templates/hello_world --data=templates/hello_world/data.json
+````
+
+When finished Nota has rendered a very simple PDF page, consisting of nothing much.
+Try it yourselves with any of the examples provided.
+
+You can use any inline CSS or linked stylesheets.
+Any of the images you use should be available in the template directory.
+
+Add the switch `--template=<dir>` to select a template by directory.
+Add the switch `--data=<path>` with a path from the Nota root directory to the JSON that should be rendered.
+Add the switch `--port=<port>` with a port (larger than 1024) to select which port to use. This is useful for situations in which you are rendering lots of PDFs simultaneously.
+
+By default Nota will output the PDF in the root rolder of itself, in a file called `output.pdf`.
+When this is not want you want, simply add `--output=x.pdf` (which will save the file in the Nota root folder) or `--output=/tmp/x.pdf` (which saves the file on the absolute path).
+
+
+## Known problems
+
+Nota is young, experimental, and built on a still developing tech stack. There are still quite some shortcomings and bugs (none that aren't likely to be fixed in the near future). That said, we've been able to use Nota in production environments already. If you take care to test your setup, verify the results and can live with some flaws, you should be fine. Consider the current version a showcase of the potential of this tech stack and the future of Nota. Here's some things to take into account:
+
+#### No clickable hyperlinks
+Even though WebKit supports this, due to a [bug](https://github.com/ariya/phantomjs/issues/10196) in QtWebKit which PhantomJS builts on the current output PDFs have no clickable links. This has quite some attention, but no confirmed fix yet it seems.
+
+#### Selectable text
+It seems PhantomJS only generates PDFs with selectable text on Linux due to a [bug](https://github.com/ariya/phantomjs/issues/10373). More information/research on specifics is needed.
+
+#### Bad kerning
+Under Linux the output PDF's text has bad kerning. See [this reported issue](https://github.com/ariya/phantomjs/issues/12016). More information/research on specifics is needed.
+
+#### Fonts
+Due to [a bug](http://arunoda.me/blog/phantomjs-webfonts-build.html) in PhantomJS, the loading of webfonts (even if they're locally hosted) seems broken. For now you'll have to install the fonts on the system manually, and then they'll load as expected. More research on in development versions of PhantomJS is needed.
+
+
+## Meta
+
+#### Developers
+Nota is developed by Aerix, an small internetbureau  and inventid, two Dutch 
```

Comment:
Haha, wups jup, I think I got distracted and then the task to get back to it vanished from the stack :) Any ideas?